Package.swift

```diff
@@ -10,9 +10,8 @@ let package = Package(
         .library(name: "GDSCommon", targets: ["GDSCommon"])
     ],
     dependencies: [
-        .package(
-            url: "https://github.com/nalexn/ViewInspector",
-            .upToNextMajor(from: "0.9.7"))
+        .package(url: "https://github.com/nalexn/ViewInspector",
+                 .upToNextMajor(from: .init(0, 10, 1)))
```

Comment:
should we be pinning this to a specific git commit hash?

Reply:
Sure, I guess it's maybe a bigger discussion and not worth deviating for this one on its own. Reason for doing it is that it helps protect against a situation where a dependency is taken over by a malicious actor.

Reply:
Yeah I think maybe a bigger conversation around our packages, so far we have been following the above :)